astropy/modeling/utils.py

```diff
@@ -484,3 +484,59 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def render_models2sky(model_list, image):
+    """
+    Evaluates a list of 2D models on an 2D image array. Limits the evaluation to the
+    bounding box limits if present. 
+    """
+    #from .. nddata.utils import add_array
+
+    # Get position param names if not input.
+    # Try known variants, throw an error if not reconized.
+    # Better way to do this?
+    # Deal with this later, considering only Gaussian2D for now.
+    # for pos_key in ('mean', 'pos', '0'):
+    #     try:
+    #         x_pos = model.parameters[model.param_names.index('x_' + pos_key)]
+    #         y_pos = model.parameters[model.param_names.index('y_' + pos_key)]
+    #         break
+    #     except:
+    #         pass
+
+    # try:
+    #     x_pos, y_pos
+    # except:
+    #     raise InputParameterError, 'position keywords not found.'
+    
+    image = image.copy()
+    im_shape = image.shape
+
+
+    for model in model_list:
+
+        try:
+
+            bbox = model.bbox
+            assert bbox is not None
+
+            bbox = (max([0, bbox[0]]), min([im_shape[1]-1, bbox[1]]) + 1,
+                               max([0, bbox[2]]), min([im_shape[0]-1, bbox[3]]) + 1)
+
+            x, y = np.meshgrid(np.arange(bbox[0],bbox[1]).astype(int),
```

Comment:
Changing to [np.ogrid](http://docs.scipy.org/doc/numpy/reference/generated/numpy.ogrid.html) should work just as well, but in common cases use much less memory?